wiki/Play_Styles/en.md

```diff
@@ -1,165 +1,169 @@
+
+*Note: Since the pen tap was considered as `M1`, only the secondary tap button will work, and `X` is usually the default secondary tap button.*
 
 ### Tablet + Keyboard
 
-A popular play style for players with a graphics tablet (Digitizer).
+*Notice: This play style requires the use of a graphics tablet (Digitizer).*
 
-Tablet + Keyboard players use a tablet pen instead of a mouse to navigate the in-game cursor and tap beats using the keyboard or the tablet buttons.
-Much like Mouse + Keyboard, some players also use only the keyboard to tap while others will switch to keyboard when streaming.
+Tablet + Keyboard is a popular play style among many passionate, competitive, and top players.
 
-Sometimes referred to as _tapx_, it describe how the streaming with Tablet-only playing works.
-To stream as a Tablet-only player, the player should alternate between tapping the pen on the tablet and pressing one assigned secondary key (since the pen tap was considered as `M1`, only the secondary tap button will work, and `X` is usually the default secondary tap button).
+Tablet + Keyboard players "hover" and/or "drag" a tablet pen to navigate the in-game cursor and tap beats using the keyboard or the tablet buttons.
+
+Much like Mouse + Keyboard, some players also use only the tablet pen to tap while others will switch to keyboard when streaming.
 
 ### Touchscreen
 
-A possible play style for players who own a touch-enabled monitor, laptop screen or tablet.
+This play style uses a touch-enabled monitor, laptop screen, or tablet to navigate the in-game cursor and tap hit objects.
 
-Players touch the screen to instantaneously navigate and tap to the beat.
-This play style may feel slightly similar to playing [osu!stream](/wiki/osu!stream) by touching the screen directly rather than by hardware support (mouse, graphic tablet).
-Some players may opt for this play style when available or when they do not own a graphic tablet.
+To navigate the in-game cursor, touchscreen players touch the screen of their device with a tablet pen or their finger at the precise location in which they want the cursor to be at. This results in the cursor to appear as if it is "jumping" to the touched location.
 
-The key overlay shows touch input as `M1`.
+When a touchscreen player taps the screen, their device will simultaneously move the cursor to that point, as well as send a mouse input. Thus, to tap beats, follow sliders, and spin spinners, touchscreen players will only touch hit objects when they want the hit object to be hit.
 
-### Touchscreen + Keyboard
+This play style may feel slightly similar to playing [osu!stream](/wiki/osu!stream) due to the act of touching the screen directly rather than by hardware support (e.g., mouse, graphics tablet).
```

Comment:
```suggestion
This play style may feel similar to playing [osu!stream](/wiki/osu!stream) due to the act of touching the screen directly rather than moving a cursor.
```